{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py

```diff
@@ -249,6 +260,10 @@ def wrapped(*args, **kwargs):
 }
 {%- endif %}
 
+{%- if cookiecutter.monitoring == "y" %}
+REDIS_URL = env("REDIS_URL", default="")
```

Comment:
```suggestion
REDIS_URL = env("REDIS_URL")
```

Reply:
actually, this is silly

why aren't we using REDIS_HOST and REDIS_PORT ?

Reply:
that wasn't the forbidden part, please change it to:
```
REDIS_HOST = env("REDIS_HOST")
REDIS_PORT = env.int("REDIS_PORT")
REDIS_URL = f"redis://{REDIS_HOST}:{REDIS_PORT}"
```

sorry for not being clear in the beginning

Reply:
thank you and sorry again for confusion